perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -198,8 +221,29 @@ def PrepareVMEnvironment(self):
       if self.is_static:
         self.SnapshotPackages()
       self.SetupPackageManager()
+    self.ConfigureVMKernel()
     self.BurnCpu()
 
+  def ConfigureVMKernel(self):
+    """Apply {procfs,sysfs}_config to VM."""
+
+    path_components = None
+
+    def ConfigureSubtree(subtree):
+      for name, value in subtree.iteritems():
+        path_components.append(name)
+        if not isinstance(value, dict):
+          self.RemoteCommand('echo "%s" | sudo tee %s' %
+                             (str(value), posixpath.join(*path_components)))
```

Comment:
nit: you shouldn't have to do str(x) as the %s will do that for you.  But no need to change just for that.